libs/angular-accelerator/src/lib/components/page-header/page-header.component.spec.ts

```diff
@@ -115,13 +115,14 @@ describe('PageHeaderComponent', () => {
 
   it('should check permissions and render buttons accordingly', async () => {
     expect(await pageHeaderHarness.getInlineActionButtons()).toHaveLength(0)
-    expect(await pageHeaderHarness.getOverflowActionButtons()).toHaveLength(0)
+    expect(await pageHeaderHarness.getOverflowActionButton()).toBeNull()
```

Comment:
If this is the button to open the overflow menu you should call it getOverflowActionMenuButton or something like that